packages.yml

```diff
@@ -1,5 +1,10 @@
 packages:
-- package: fivetran/shopify
-  version: [">=0.10.0", "<0.11.0"]
+# - package: fivetran/shopify
+#   version: [">=0.11.0", "<0.14.0"]
+
+- git: https://github.com/fivetran/dbt_shopify.git
+  revision: feature/standardized-line-item-model
+  warn-unpinned: false
+
```

Comment:
Also, I think we should regenerate the docs so the most recent shopify models get picked up.

Reply:
Thanks Catherine!